etc/cspell.config.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+if (process.env.GITHUB_WORKSPACE) {
```

Comment:
How about?:
```
const BASEPATH = process.env.GITHUB_WORKSPACE || '.'
```

Reply:
See https://github.com/SeisoLLC/goat/pull/4/commits/0ee502a0bfd52e1eb46ab9e9375f5900c28cf028; I replaced it with a ternary